static/usage/breadcrumbs/collapsing-items/popover-on-click/demo.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Popover</title>
+  <link rel="stylesheet" href="../../../common.css" />
+  <script src="../../../common.js"></script>
+  <script type="module" src="https://cdn.jsdelivr.net/npm/@ionic/core@6/dist/ionic/ionic.esm.js"></script>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@ionic/core@6/css/ionic.bundle.css" />
+</head>
+
+<body>
+  <ion-app>
+    <ion-content>
+      <div class="container">
+        <ion-breadcrumbs max-items="4">
+          <ion-breadcrumb href="#home">Home</ion-breadcrumb>
+          <ion-breadcrumb href="#electronics">Electronics</ion-breadcrumb>
+          <ion-breadcrumb href="#photography">Photography</ion-breadcrumb>
+          <ion-breadcrumb href="#cameras">Cameras</ion-breadcrumb>
+          <ion-breadcrumb href="#film">Film</ion-breadcrumb>
+          <ion-breadcrumb href="#35mm">35 mm</ion-breadcrumb>
+        </ion-breadcrumbs>
+      </div>
+    </ion-content>
+  </ion-app>
+
+  <script>
+    const breadcrumbs = document.querySelector('ion-breadcrumbs');
+    breadcrumbs.addEventListener('ionCollapsedClick', presentPopover);
+
+    class PopoverPage extends HTMLElement {
+      constructor() {
+        super();
+      }
+
+      connectedCallback() {
+        let breadcrumbTemplate = ``;
+        this.collapsedBreadcrumbs.forEach(breadcrumb => {
+          breadcrumbTemplate += `
+            <ion-item href="${breadcrumb.href}">
```

Comment:
Can we remove the line from the last item if it's easy to do? 
![image](https://user-images.githubusercontent.com/2721089/172652754-b1cea2dd-7942-461e-92f0-5c8b67c199c2.png)

Reply:
Is it possible to also use an inline popover here instead? Because the demo is lazy loaded, there's a delay between when you click the collapsed indicator and when the popover opens. Using the inline popovers means the popover is already available.

Same goes for the per-framework examples.